### PR TITLE
i.sentinel.download: Run sys.exit() after message "No product found"

### PR DESCRIPTION
**Pull request by nifg414**

### CODE REVIEW

File: src/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py [hunk 1]
```diff
@@ -667,7 +667,8 @@ def filter(
         products_df = self._api.to_dataframe(products)
         if len(products_df) < 1:
             gs.message(_("No product found"))
-            return
+            sys.exit()
+
```
Comment (ninsbl) [inline src/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py]:
```suggestion
            sys.exit()
```

**Review by ninsbl — COMMENTED** [review]
Required change to satisfy black

**Comment by nifg414** [comment]
i.sentinel.download tested locally and works as expected.

**Review by ninsbl — APPROVED** [review]

**Comment by wenzeslaus** [comment]
I'm late, but that's what `gs.fatal(...)` is for. User then sees `ERROR:` (signaling that further operation is not possible) and receives non-zero return code (useful for scripting).

**Comment by ninsbl** [comment]
Right, sorry. I should have kept the PR open a bit longer for discussion.
The reason `sys.exit()` appeared right to me is that I would not consider it an error if querying the Copernicus database returns an an empty result. The previous code implied the same line of thinking (message + return)...

**Pull request merged**